src/components/Thumbnail/Thumbnail.tsx

```diff
@@ -18,75 +19,70 @@ type ThumbnailProps = {
   links?: Link[];
 };
 
-const defaultEasing = [0.6, -0.05, 0.01, 0.99];
-
 export function Thumbnail({ title, subTitle, img, description, links }: ThumbnailProps) {
   return (
     <m.article
       css={articleCss}
-      initial="default"
-      whileHover="hover"
-      animate="default"
-      variants={articleVariants}
+      variants={defaultFadeInVariants}
+      initial="initial"
+      animate="animate"
+      exit="exit"
     >
-      <m.div css={imageCss} variants={imageVariants}>
-        <Image src={img} alt={title} fill quality={100} />
-      </m.div>
-      <m.div css={contentsCss}>
-        <m.div>
-          <m.p css={titleCss} variants={textVariants}>
-            {title}
-          </m.p>
-          <m.p css={subTitleCss} variants={textVariants}>
-            {subTitle}
-          </m.p>
-        </m.div>
-        <m.p
-          css={descriptionCss}
-          variants={textVariants}
-          dangerouslySetInnerHTML={{ __html: description }}
-        />
+      <Image css={imageCss} src={img} alt={title} fill quality={100} />
+      <div css={gradientCss} />
+      <div css={contentsCss}>
+        <div>
+          <p css={titleCss}>{title}</p>
+          <p css={subTitleCss}>{subTitle}</p>
+        </div>
+        <p css={descriptionCss} dangerouslySetInnerHTML={{ __html: description }} />
         {links && (
-          <m.div css={linkContainerCss}>
+          <div css={linkContainerCss}>
             {links.map(link => (
-              <m.span key={link.type} css={linkWrapperCss}>
-                <m.a href={link.href} css={linkCss} variants={textVariants}>
+              <span key={link.type} css={linkWrapperCss}>
+                <a href={link.href} css={linkCss}>
```

Comment:
새로운 탭으로 열리는게 더 사용성 좋다고 생각하는데 어떻게 생각하시나요??

```suggestion
                <a href={link.href} target="_blank" css={linkCss}>
```

Reply:
오 좋은것 같아요 수정할게요 !